src/frontend/src/lib/components/send/Send.svelte

```diff
@@ -3,10 +3,11 @@
 	import SendModal from '$lib/components/send/SendModal.svelte';
 	import { modalSend } from '$lib/derived/modal.derived';
 	import { modalStore } from '$lib/stores/modal.store';
+	import { token } from '$lib/stores/token.store';
 
 	export let isTransactionsPage: boolean;
 </script>
 
 <SendButtonWithModal open={modalStore.openSend} isOpen={$modalSend}>
-	<SendModal {isTransactionsPage} on:nnsClose slot="modal" />
+	<SendModal {isTransactionsPage} on:nnsClose slot="modal" targetNetwork={$token?.network} />
```

Comment:
Did you faced an issue? In which use case?

Spontaneously, I would think that we might omit `targetNetwork` here on purpose but, of course I might be wrong therefore asking about the issue.

Reply:
The difference is to have two networks or one.

Without:
![Screenshot 2024-09-26 at 14 46 41](https://github.com/user-attachments/assets/f034962a-da16-4766-8045-83562c0cac70)

With:
![Screenshot 2024-09-26 at 14 48 17](https://github.com/user-attachments/assets/96cca397-6ed7-42dd-b8c1-d443f46303f4)

Rethinking about it, I wonder whether just one makes more sense.

Should I remove the prop which is not used anyway? It seems useful, but maybe more for conversions and maybe they use another modal now?

Reply:
I get the feeling that the destination network is only displayed or provided if it's different, like ckETH -> ETH.

That's why the first screenshot says "Network" while the second says "Source / Destination Network".

Given that we have no tests, we should be extra cautious if we modify anything. However, if you've encountered an error (not cosmetic), we definitely need to investigate.

Reply:
No error. I'll close this PR.